src/lib/converter/nodes/class.ts

```diff
@@ -36,7 +44,15 @@ export class ClassConverter extends ConverterNodeComponent<ts.ClassDeclaration>
         context.withScope(reflection, node.typeParameters, () => {
             if (node.members) {
                 node.members.forEach((member) => {
-                    this.owner.convertNode(context, member);
+                    let privateMember = (member.flags & 16) > 0;
```

Comment:
Hi Blake, made some updates inline with your suggestions for clarity. Take a look...